From be38e33f218ea95865847273486ad051a3377593 Mon Sep 17 00:00:00 2001
From: Yuchou Gan <yuchou.gan@nxp.com>
Date: Fri, 30 Dec 2016 19:30:03 +0800
Subject: [PATCH 1442/5242] MGS-2460 [#ccc] 7ULP1 GPU suspend/resume problem

commit  fe3f686c20fe98443ad4d34ff2d4c8edfd32ee63 from
https://source.codeaurora.org/external/imx/linux-imx.git

Do not enable the gpu clock when initialize the clock

Date: Dec 30, 2016
Signed-off-by: Yuchou Gan <yuchou.gan@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/clk/imx/clk-imx7ulp.c |    2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/clk/imx/clk-imx7ulp.c b/drivers/clk/imx/clk-imx7ulp.c
index 06bdd84..4051158 100644
--- a/drivers/clk/imx/clk-imx7ulp.c
+++ b/drivers/clk/imx/clk-imx7ulp.c
@@ -177,8 +177,6 @@ static void __init imx7ulp_clocks_init(struct device_node *scg_node)
 		imx_clk_prepare_enable(clks[clks_init_on[i]]);
         imx_clk_set_parent(clks[IMX7ULP_CLK_GPU2D], clks[IMX7ULP_CLK_APLL_PFD2]);
         imx_clk_set_parent(clks[IMX7ULP_CLK_GPU3D], clks[IMX7ULP_CLK_APLL_PFD2]);
-        imx_clk_prepare_enable(clks[IMX7ULP_CLK_GPU3D]);
-        imx_clk_prepare_enable(clks[IMX7ULP_CLK_GPU2D]);
 
 	pr_info("i.MX7ULP clock tree init done.\n");
 }
-- 
1.7.9.5

